### PR TITLE
Azure DevOps Integration

### PR DESCRIPTION
**Pull request by unknownskl**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+- task: SonarCloudAnalyze@1
+  displayName: Run Sonarcloud analyzer
+
+- task: PublishBuildArtifacts@1
+  displayName: Publish artifacts
+  inputs:
+    PathtoPublish: '$(Build.ArtifactStagingDirectory)'
+    ArtifactName: 'backend'
+    publishLocation: 'Container'
```
Comment (vsilvar) [inline azure-pipelines.yml:62]: This task is deprecated please use the `PublishPipelineArtifact` task instead:
```suggestion
- publish: '$(Build.ArtifactStagingDirectory)'
  artifact: 'backend'
```

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    version: '5.x'
+
+- task: SonarCloudPrepare@1
+  displayName: Prepare Sonarcloud
+  inputs:
+    SonarCloud: 'Sonarcloud'
+    organization: 'dept'
+    scannerMode: 'MSBuild'
+    projectKey: 'dtnl-livelight-website'
+    projectName: 'dtnl-livelight-website'
```
Comment (vsilvar) [inline azure-pipelines.yml:31]: These should probably come from pipeline variables.

**Review by vsilvar — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (unknownskl) [reply azure-pipelines.yml]: Thanks. Had no idea that the task was deprecated. Updated it to the new task

File: azure-pipelines.yml (hunk 2 above)
Comment (unknownskl) [reply azure-pipelines.yml]: Thanks for the suggestion. The hardcoded values has been replaced with variables.

**Review by vsilvar — APPROVED** [review]

**Pull request merged**